Apps/PackageTest/0.69.0/metro.config.js

```diff
@@ -0,0 +1,108 @@
+/**
```

Comment:
We don't need all the customizations for dealing with sym links since PackageTest is specifically for testing the built package. Looks like this got messed up in the 0.65 version of PackageTest. I'd say we should at least fix it for this file (I believe the default metro.config.js should be fine), and ideally fix it for 0.65 too, though we could do that later if needed.